pkg/framework/image_metadata.go

```diff
@@ -190,11 +193,19 @@ func ExtractPrometheusConfigurationFromImage(dockerImage *dockerv10.DockerImage)
 }
 
 // IsPersistenceEnabled verifies if the image has labels indicating that persistence is enabled
-func IsPersistenceEnabled(dockerImage *dockerv10.DockerImage) (enabled bool) {
+func IsPersistenceEnabled(name, namespace string, cli *client.Client) (enabled bool) {
+	imageStreamName := types.NamespacedName{
+		Namespace: namespace,
+		Name:      name,
+	}
+	dockerImage, err := openshift.ImageStreamC(cli).FetchDockerImage(imageStreamName)
```

Comment:
This implementation becomes OpenShift specific. Wouldn't it have more sense to keep previous function signature (with DockerImage) and retrieve the Docker image in kogitoapp_controller.go?